Directory.Build.targets

```diff
@@ -4,14 +4,14 @@
     <PackageOutputPath Condition="'$(IsProductComponent)' == 'true' ">$(ProductPackageOutputPath)</PackageOutputPath>
```

Comment:
Does this still work? Or is Arcade overriding us?

Reply:
Arcade does override us. I'm removing our attempts to do it the old way in a new commit. If we want to try to keep it the old way it seems like we'd need Arcade to stop overwriting these?

Reply:
Let's keep our layout until we can figure out what to do with our release pipeline. Our release pipeline very much depends on the layout of the artifacts folder.

You can override SDK by assigning PackageOutputPath after the Import to Arcade's Sdk.targets file.